app/templates/password/reset.hbs

```diff
@@ -0,0 +1,3 @@
+{{password/reset-password 
+  resetPassword=(route-action "resetPassword")
```

Comment:
I think `route-action`'s are less testable in component tests, though you can correct me if I'm wrong on this. Also fairly certain that you don't need it here if the action is on the route, and similarly wonder if it shouldn't be in a controller.